package.json

```
@@ -23,6 +23,8 @@
     "i18n:check": "grunt checktextdomain",
     "i18n": "npm run -s i18n:js && npm run -s i18n:php && npm run -s i18n:check && npm run -s i18n:pot",
     "lint": "npm run lint:js",
```

Maybe for another PR, but shouldn't `lint` run all linters?
```suggestion
     "lint": "npm run lint:js && npm run lint:css && npm run lint:php",
```